catalog/setup-jest.ts

```diff
@@ -11,3 +11,5 @@
   mixpanelToken: '',
   passwordAuth: 'ENABLED',
 }
+;(window as any).TextEncoder = require('util').TextEncoder
```

Comment:
why do we need this? pls add a comment

Reply:
"It's required for jsdom" - is that enough? I don't know real reason why is it required for jsdom, and why it doesn't work out of the box even we set `"testEnvironment": "jsdom"`